src/main/java/org/cryptomator/frontend/fuse/ReadOnlyAdapter.java

```diff
@@ -320,7 +317,7 @@ public void close() throws IOException {
 	 * Attempts to get a specific error code that best describes the given exception.
 	 * As a side effect this logs the error.
 	 *
-	 * @param e An exception
+	 * @param e      An exception
```

Comment:
unrelated

Reply:
done in https://github.com/cryptomator/fuse-nio-adapter/pull/80/commits/a4e330bb0b8154847c9ee048f01b2788ad052a25